runtime/rococo/src/propose_parachain.rs

```diff
@@ -338,6 +363,10 @@ impl<T: Config> pallet_session::SessionManager<T::ValidatorId> for Module<T> {
 			validators.extend(proposal.validators);
 		}
 
+		let queued = ValidatorsToAdd::<T>::take();
+		validators.extend(queued);
+		validators.dedup();
```

Comment:
dedup only works on sorted sets, I think you need to call `validators.sort();` before
```suggestion
		validators.sort();
		validators.dedup();
```